db/schema.rb

```diff
@@ -44,6 +54,7 @@
     t.string   "last_sign_in_ip"
     t.datetime "created_at"
     t.datetime "updated_at"
+    t.string   "name"
   end
 
   add_index "users", ["email"], name: "index_users_on_email", unique: true
```

Comment:
Not the best with devise + omniauth but I took a look at some other examples and this seems to follow their flow. Thanks for doing this :+1: